academy/xl-cosmjs/first-steps.md

```diff
@@ -0,0 +1,637 @@
+---
+title: "Bank - Send Tokens"
+order: 2
+description: Interacting with a Cosmos SDK chain through CosmJs
+tag: deep-dive
+---
+
+# Bank - Send Tokens
+
+Now that you know what CosmJs is, you should take your first steps in using it. A basic feature of a Cosmos chain is the ability to send tokens via the `bank` module. CosmJs naturally offers functions to cover this facility. You are going to:
+
+1. Use an existing test net with a key of your own.
```

Comment:
I think the jargon is to use [`testnet`](https://en.wikipedia.org/wiki/Testnet) in one word, or to say `test network`.